### DISCUSSION TITLE
Is it possible to automatically update the state of components by some webhooks?

### DISCUSSION THREAD

**Discussion by palicepta** [Status-Page/Status-Page#599] [Q&A] [answered]
subj.

**Comment by herrtxbias** [comment, marked as answer]
You can only use the API to update the state of components.

**Comment by palicepta** [comment]
Hi [herrtxbias](https://github.com/herrtxbias),
Could you provide a few examples of how to change a component's status using an API request?

Thank you in advance!

**Comment by palicepta** [comment]
I tried to request:
curl -X 'GET'   'https://status.domain.com/api/components/component-groups/'   -H 'accept: application/json'   -H 'Authorization: f38666e24\*\*\*cbbb'   -H 'X-CSRFToken: PrUrHkd3pbSd*\*\*Sm1YLSjO5RjgBEW'

And got error like this, could you help me?
{"detail":"Authentication credentials were not provided."}

**Reply by herrtxbias** [reply]
You need to prefix the actual token in your authorization header with the Token Type, in this case it's "Token". A correct Header would look like this: `Authorization: Token f38666e24***cbbb`

**Reply by palicepta** [reply]
Thank you for your reply
I took information from rest API documentation and here is the curl request without "Token"
<img width="1061" alt="image" src="https://github.com/Status-Page/Status-Page/assets/112872766/3f1e3eca-65e4-4c6a-9e0b-69415970294f">
